specs/pvs/corla.pvs

```diff
@@ -922,8 +995,10 @@ END cast_vote_records
 %%% the county-level, and one is for audit board members.
```

Comment:
It may be appropriate to describe the public dashboard here. However, unlike the other interfaces here, it is only "interactive" in a superficial way: public users cannot modify any data in the RLA system, they are just presented with read-only views of the data. It is still fine to specify _what_ they are presented with, but @kiniry may prefer to spec that elsewhere?